Clarify error messages by specifying WebFinger handle format

obenland opened the pull request:
See https://wordpress.org/support/topic/make-follow-reply-button-form-easier-for-users/#post-18620743

## Proposed changes:
* Update error messages in follow-me and remote-reply blocks to specify "WebFinger handle" instead of just "handle"
* Change `invalidProfileError` message from "valid profile URL or handle" to "valid profile URL or WebFinger handle"
* Applies to both src and build directories

### Other information:

- [ ] Have you written new tests for your changes, if applicable?

This change addresses user feedback from [WordPress.org support forum](https://wordpress.org/support/topic/make-follow-reply-button-form-easier-for-users/#post-18620743) requesting clearer guidance about the expected input format.

## Testing instructions:

* Go to a post with ActivityPub follow-me or remote-reply blocks enabled
* Try to enter an invalid handle format (e.g., just "mastodon.social")
* Verify the error message now shows "Please enter a valid profile URL or WebFinger handle" instead of the previous generic "handle" text
* Confirm this appears in both follow-me and remote-reply block error states

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Added - for new features
-   [x] Changed - for changes in existing functionality
-   [ ] Deprecated - for soon-to-be removed features
-   [ ] Removed - for now removed features
-   [ ] Fixed - for any bug fixes
-   [ ] Security - in case of vulnerabilities

#### Message

Clarify error messages by specifying WebFinger handle format for better user guidance

</details>

obenland committed:
Clarify error messages by specifying WebFinger handle format

Update error text from "handle" to "WebFinger handle" to provide users
with clearer guidance about the expected format (@user@server.com).

Addresses user feedback from WordPress.org support forum requesting
more specific guidance for follow/reply form inputs.

matticbot committed:
Add changelog

pfefferle commented:
Is "WebFinger" something a regular user understands? I would assume not!?

obenland commented:
What would you suggest?

pfefferle commented:
I actually like "handle"... or "email like" handle?

Have you tried to ask chatgpt?

obenland commented:
> Please enter a valid profile URL or use the `@username@server` format.

We do show `@username@example.com` as the placeholder in that input field.

pfefferle commented:
Yes, we do!

pfefferle commented on build/follow-me/render.php:
```diff
@@ -49,7 +49,7 @@
 			'emptyProfileError'   => __( 'Please enter a profile URL or handle.', 'activitypub' ),
 			'genericError'        => __( 'An error occurred. Please try again.', 'activitypub' ),
-			'invalidProfileError' => __( 'Please enter a valid profile URL or handle.', 'activitypub' ),
+			'invalidProfileError' => __( 'Please enter a valid profile URL or WebFinger handle.', 'activitypub' ),
```
```suggestion
			'invalidProfileError' => __( 'Please enter a valid profile URL or handle.', 'activitypub' ),
```

pfefferle commented on build/remote-reply/render.php:
```diff
@@ -33,7 +33,7 @@
 			'emptyProfileError'   => __( 'Please enter a profile URL or handle.', 'activitypub' ),
 			'genericError'        => __( 'An error occurred. Please try again.', 'activitypub' ),
-			'invalidProfileError' => __( 'Please enter a valid URL or handle.', 'activitypub' ),
+			'invalidProfileError' => __( 'Please enter a valid profile URL or WebFinger handle.', 'activitypub' ),
```
```suggestion
			'invalidProfileError' => __( 'Please enter a valid profile URL or handle.', 'activitypub' ),
```

pfefferle commented on src/follow-me/render.php:
```diff
@@ -49,7 +49,7 @@
 			'emptyProfileError'   => __( 'Please enter a profile URL or handle.', 'activitypub' ),
 			'genericError'        => __( 'An error occurred. Please try again.', 'activitypub' ),
-			'invalidProfileError' => __( 'Please enter a valid profile URL or handle.', 'activitypub' ),
+			'invalidProfileError' => __( 'Please enter a valid profile URL or WebFinger handle.', 'activitypub' ),
```
```suggestion
			'invalidProfileError' => __( 'Please enter a valid profile URL or handle.', 'activitypub' ),
```

pfefferle commented on src/remote-reply/render.php:
```diff
@@ -33,7 +33,7 @@
 			'emptyProfileError'   => __( 'Please enter a profile URL or handle.', 'activitypub' ),
 			'genericError'        => __( 'An error occurred. Please try again.', 'activitypub' ),
-			'invalidProfileError' => __( 'Please enter a valid URL or handle.', 'activitypub' ),
+			'invalidProfileError' => __( 'Please enter a valid profile URL or WebFinger handle.', 'activitypub' ),
```
```suggestion
			'invalidProfileError' => __( 'Please enter a valid profile URL or handle.', 'activitypub' ),
```

obenland committed:
Update build/follow-me/render.php

Co-authored-by: Matthias Pfefferle <pfefferle@users.noreply.github.com>

obenland committed:
Update build/remote-reply/render.php

Co-authored-by: Matthias Pfefferle <pfefferle@users.noreply.github.com>

obenland committed:
Update src/follow-me/render.php

Co-authored-by: Matthias Pfefferle <pfefferle@users.noreply.github.com>

obenland committed:
Update src/remote-reply/render.php

Co-authored-by: Matthias Pfefferle <pfefferle@users.noreply.github.com>

pfefferle reviewed (commented):
Please also update the changelog text.

obenland committed:
Clarify error messages in account modal

Update error messages to specify the required full profile URL format, improving user guidance in the account modal.
